src/kakarot/instructions/logging_operations.cairo

```diff
@@ -39,6 +39,13 @@ namespace LoggingOperations {
 
         // Pop offset + size.
         let (stack, popped) = Stack.pop_n(stack, topics_len + 2);
+        let ctx = ExecutionContext.update_stack(ctx, stack);
+
+        // This instruction is disallowed when called from a `staticcall` context, which we demark by a read_only attribute
+        if (ctx.read_only == 1) {
```

Comment:
Should we raise here? Not sure

Reply:
Thinking about it, not raising means that the success/failure result of the callops can't be depended on. The state won't be updated for the calls, but the return of the call would always be one. 

I think it makes sense to raise.